### PR TITLE
Issue 1: Initialize repo and create basic administrative documents

### LINKED ISSUE #1: Initialize repo and create basic administrative documents
**Description:**

- Complete administrative tasks such as initialize the repo and create basic administrative documents, such as a README containing project description, tech stack, group members, etc. 
- Create relevant milestones and labels for use in future issues.

### PR DESCRIPTION
**Pull request by ZackHolmberg**
- Completed administrative tasks such as initializing the repo and create basic administrative documents, such as a README containing project description, tech stack, group members, etc. Additionally, created relevant milestones and labels for use in future issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+- With the BisonCoin app, students will be able to send and receive to their peers, as well as businesses in order to purchase products.
+- Students will be able to use the app to view how much BisonCoin they have in their digital wallet, as well as view their transaction history.
+- Students will be able to act as miners on the blockchain network, earning currency as a reward.
+
+## Tech Stack
+
+- **Blockchian:** `Python`
+- **Micro-servies:** `Python`
+- **Front-end Web app:** `Vue.js, Node.js or Flask` (TBD)
+- **Data Storage:** `MongoDB`, if needed
```
Comment (akshaysharma21) [inline README.md]: If I'm not mistaken, we will definitely be needing the database to store user info. we should remove if needed in that case.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+
+## Architecture Diagram
+
+![Architecture](assets/Architecture.png)
```
Comment (abhisachdev17) [inline README.md]: This can go in wiki, No need for this to be in Readme I think

**Review by abhisachdev17 — APPROVED** [review]

**Review by akshaysharma21 — CHANGES_REQUESTED** [review]
Can we remove the "if needed" from the mongo db stuff? the rest looks good to me

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+
+- BisonCoin is a new cryptocurrency intended for use by University of Manitoba students to exchange on campus between each other or a variety of products at all campus businesses, including all food outlets and the book store!
+
+- The system will be composed of:
+  - A front-end web application
+  - A collection of microservices
+  - A Blockchain, cryptocurrency implementation
+  - Data storage, if needed
+
+## Core Features
```
Comment (madisonfines) [inline README.md]: Maybe add a managing account feature as well since that is considered one of our core features

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+
+- With the BisonCoin app, students will be able to send and receive to their peers, as well as businesses in order to purchase products.
```
Comment (madisonfines) [inline README.md]: Make it clear that they are sending and receiving "BisonCoin"

File: README.md [hunk 5]
```diff
@@ -1,2 +1,37 @@
...
+- Students will be able to use the app to view how much BisonCoin they have in their digital wallet, as well as view their transaction history.
+- Students will be able to act as miners on the blockchain network, earning currency as a reward.
+
+## Tech Stack
```
Comment (madisonfines) [inline README.md:29]: Typos for "Blockchain" and "Microservices"

File: README.md (hunk 4 above)
Comment (madisonfines) [reply README.md]: Also, I don't think we need the first part saying "with the BisonCoin app"

**Review by madisonfines — CHANGES_REQUESTED** [review]
A few small changes, otherwise looks good!

File: README.md [hunk 6]
```diff
@@ -1,2 +1,37 @@
...
+
+- **Blockchian:** `Python`
+- **Micro-servies:** `Python`
+- **Front-end Web app:** `Vue.js, Node.js or Flask` (TBD)
```
Comment (abhisachdev17) [inline README.md]: This should only be Vue.js. Flask should go in the Microservices section.

File: README.md (hunk 2 above)
Comment (ZackHolmberg) [reply README.md]: Good point! Removed.

File: README.md (hunk 1 above)
Comment (ZackHolmberg) [reply README.md]: Yep good point, updated!

File: README.md (hunk 4 above)
Comment (ZackHolmberg) [reply README.md]: Great points, updated!

File: README.md (hunk 6 above)
Comment (ZackHolmberg) [reply README.md]: Updated

File: README.md (hunk 5 above)
Comment (ZackHolmberg) [reply README.md:29]: Oops. Updated!

File: README.md (hunk 6 above)
Comment (abhisachdev17) [reply README.md]: I uploaded some roadmap and user story assets for this PR. Make sure to merge and push

File: README.md [hunk 7]
```diff
@@ -1,2 +1,33 @@
...
+## Core Features
+
+- Students will be able to send and receive BisonCoin to their peers, as well as businesses in order to purchase products.
+- Students will be able to use the app to view how much BisonCoin they have in their digital wallet, as well as view their transaction history.
+- Students will be able to act as miners on the blockchain network, earning currency as a reward.
+
+## Tech Stack
+
+- **Blockchain:** `Python`
+- **Microservices:** `Python`
```
Comment (abhisachdev17) [inline README.md]: Flask in here

File: README.md [hunk 8]
```diff
@@ -1,2 +1,33 @@
...
+- **Front-end Web app:** `Vue.js & potentially Node.js` (TBD)
```
Comment (abhisachdev17) [inline README.md]: Why would we use Node for our front end?

File: README.md (hunk 3 above)
Comment (ZackHolmberg) [reply README.md]: You're 100% right. Updated!

File: README.md (hunk 7 above)
Comment (ZackHolmberg) [reply README.md]: Flask is just a Python library so I don't know if we need to specify it, that's my thoughts at least. Lmk what you think

File: README.md (hunk 8 above)
Comment (ZackHolmberg) [reply README.md]: I'm unsure how we will only use Vue in order to send requests to the microservices; Node might make things easier, but I figured we can finalize that post Sprint 1.

File: README.md (hunk 7 above)
Comment (abhisachdev17) [reply README.md]: Wouldn't hurt to specify, example repo specified Node and Express as well.

**Review by madisonfines — APPROVED** [review]

**Review by akshaysharma21 — APPROVED** [review]

**Pull request merged**